exercises/B-boolean-literals/exercise.js

```diff
@@ -4,9 +4,10 @@
   This program needs some variables to log the expected result.
   Add the required variables with the correct boolean values assigned.
 */
-
 var codeYourFutureIsGreat = true;
-
+let mozafarIsCool = false;
```

Comment:
Great that you have chosen not to use "var"

Reply:
Good job Alex I take it for next time